packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -959,6 +979,14 @@ export class GarbageCollector implements IGarbageCollector {
 			headers: headerData,
 		});
 
+		// Any time we log a Tombstone Loaded error (via Telemetry Tracker),
+		// we want to also trigger autorecovery to avoid the object being deleted
+		// Note: We don't need to trigger on "Changed" because any change will cause the object
+		// to be loaded by the Summarizer, and auto-recovery will be triggered then.
+		if (isTombstoned && reason === "Loaded") {
+			this.triggerAutoRecovery(nodePath);
```

Comment:
This may end up generating multiple ops for the same data store - one for the data store + one for each of its DDS that gets loaded. From the telemetry today, it won't create a lot of ops but maybe worth a comment in case this becomes an issue and someone is looking into it?

Reply:
The problem becomes worse if multiple clients all loading the same data store / DDS all send these ops.